Terminal.Gui/Views/ComboBox.cs

```diff
@@ -490,11 +490,11 @@ private void HideList ()
             OnOpenSelectedItem ();
         }
 
-        Rectangle rect = _listview.BoundsToScreen (_listview.IsInitialized ? _listview.Bounds : Rectangle.Empty);
         Reset (true);
-        _listview.Clear (rect);
+        _listview.Clear (_listview.IsInitialized ? _listview.Bounds : Rectangle.Empty);
         _listview.TabStop = false;
         SuperView?.SendSubviewToBack (this);
+        Rectangle rect = _listview.BoundsToScreen (_listview.IsInitialized ? _listview.Bounds : Rectangle.Empty);
```

Comment:
Hmmmm.

This is making me wonder if BoundsToScreen is perhaps a better place for that conditional.

Calls to it would seem to always need that check, no? Plus, it would clean these up and ensure consumers don't make the mistake of leaving it out.

Reply:
This is old code. I think the check can be removed.